src/utils/api.js

```diff
@@ -4,8 +4,15 @@ const myApi = axios.create({
   baseURL: "https://shrel-news.herokuapp.com/api",
 });
 
-export const fetchArticles = () => {
-  return myApi.get("/articles").then((res) => {
+export const fetchArticles = (topic) => {
+  const articleUrl = topic ? `articles?topic=${topic}` : "/articles";
```

Comment:
This works fine but you might want to make use of axios params especially when we add additional conditions. The notes are here: https://notes.northcoders.com/courses/js-front-end/react-projects